src/components/Toc/Toc.scss

```diff
@@ -0,0 +1,17 @@
+@use '../variables';
+
+$block: '.#{variables.$ns}toc';
+
+#{$block} {
+    &__title {
+        font-size: var(--g-text-body-2-font-size);
+        font-weight: 500;
```

Comment:
Should use one of typography [mixins](https://github.com/gravity-ui/uikit/blob/13ed356b1f8eaf82816f0f6e0ec3f6437b364c41/styles/mixins.scss#L149), font properties can be configured in theme and must not be hardcoded